### ISSUE TITLE
Reaction Commerce integrating a CMS

### ISSUE THREAD

**Issue by pomaking** [alisalaah/meteorcms#19]
You're doing some nice work here.  Reminds me a lot of what they're doing over at PencilBlue.  They're looking at integrating a CMS package with the Reaction Commerce platform...thought I'd point it out in case you were interested. 
https://github.com/reactioncommerce/reaction/issues/148

**Comment by alisalaah** [comment]
If I can find similar end goals then we could combine efforts.  What is
mainly needed is an admin back end that allows plugins properly and
database driven pages.  If the core is built properly to be expanded upon
that everything else will be easy.

**Comment by pomaking** [comment]
Yeah...just thought I'd point it out to you.  Not sure what works best with
their backend, but I like the simplicity of what you're doing here.

**Comment by alisalaah** [comment]
Would be nice to have free time to work on it but you know how that goes..
